contracts/plugins/assets/num/README.md

```diff
@@ -0,0 +1,33 @@
+# Collateral Plugins for nARS and snARS
+
+## Summary
+
+This plugin allows `nARS` and `snARS` holders to use their tokens as collateral in the Reserve Protocol.
+
+As described in the [Num Site](https://new.num.finance/) nTokens are ERC20 tokens, tracking the value of an underlying financial asset.
+Each nToken issued by Num Finance is fully collateralized by an asset in the traditional market. This means that for every nToken in circulation, there is a real-world asset backing it, ensuring the token's value and stability.
+
+In this particular case we're incorporating through this plugin 2 nTokens.
+
+- `nARS` is a stablecoin pegged to the `Argentine Peso (ARS)`.
```

Comment:
please include a note on the oracle used here. iirc you run the oracle and update it rather frequently. detail the % deviation and timeout thresholds (assuming they are in your documentation, but would like to have it legible directly in this repo)